packages/rancher-windows-gmsa/charts/templates/_helpers.tpl

```diff
@@ -0,0 +1,7 @@
+# Rancher
+
+{{- define "system_default_registry" -}}
+{{- if .Values.global.cattle.systemDefaultRegistry -}}
+{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
+{{- end -}}
+{{- end -}}
```

Comment:
nit: newline

Reply:
maybe we should also include these? For RKE1 / RKE2 windows support

https://github.com/rancher/charts/blob/502035b9ac9763e2ebbdde5c05f2137862bb2678/charts/rancher-monitoring/100.1.0%2Bup19.0.3/templates/_helpers.tpl#L106-L124

Reply:
We are not targeting RKE1 with GMSA support.